### PR TITLE
Remove table display on plantings#show, replace with boxes

### PR DESCRIPTION
**Pull request by Br3nda**
Change to plantings#show page to use boxes to display info, instead of an info table.
Make the most recent photo of the planting bigger, so it can be recognised by the user.

image

image

image

image

image

### CODE REVIEW

**Comment by Br3nda** [comment]
image

**Comment by Br3nda** [comment]
image

**Comment by Br3nda** [comment]
I'm going to test this with users of screen reader and other assistive technologies.

**Comment by Br3nda** [comment]
Three screen sizes

large
image

medium
image

small
image

**Comment by CloCkWeRX** [comment]
UI looks really nice, big +1 to that

File: app/helpers/plantings_helper.rb [hunk 1]
```diff
@@ -45,6 +45,13 @@ def days_from_now_to_first_harvest(planting)
     (planting.first_harvest_predicted_at - Time.zone.today).to_i
   end
 
+  def days_from_now_to_last_harvest(planting)
+    return unless planting.planted_at.present? && planting.last_harvest_predicted_at.present?
+
+    (planting.last_harvest_predicted_at - Time.zone.today).to_i
+  end
+
+
```
Comment (CloCkWeRX) [inline app/helpers/plantings_helper.rb]: Excess line break

**Comment by CloCkWeRX** [comment]
Code looks good too so far

**Comment by Br3nda** [comment]
Pull in the isotrope js library to make the layout of photos and little white boxes work nicely.

**Comment by Br3nda** [comment]
image
image

image

**Pull request merged**